snykTask/src/index.ts

```diff
@@ -88,7 +95,7 @@ function parseInputArgs(): TaskArgs {
     taskArgs.severityThreshold = taskArgs.severityThreshold.toLowerCase();
     if (isNotValidThreshold(taskArgs.severityThreshold)) {
       const errorMsg =
-        "If set, severity threshold must be 'high' or 'medium' or 'low' (case insensitive). If not set, the default is 'low'.";
+        "If set, severity threshold must be 'critical' or 'high' or 'medium' or 'low' (case insensitive). If not set, the default is 'low'.";
```

Comment:
**nit-pick (non-blocking):** We have introduced an enum. Let's use it when building the errorMsg string.

Reply:
Good idea thanks, have incorporated